docs/jaxpr.rst

```diff
@@ -162,37 +162,6 @@ before (with two input vars, one for each element of the input tuple)
   in (f,) }
 
 
-
```

Comment:
Are the constant vars truly gone? If not, then let's keep the appropriate documentation.

Reply:
I think that this is fine. However, I had to read #2 twice, because it seems to say what can be a constvar, not when they arise. How about "When JAX tracing encounters a constant it has the option to inline the constant in the jaxpr eqns, or hoist it and replace it with a constvar. Currently, Python builtin numeric types or anything with shape of () are inlined, and other array constants are replaced by constvars".

Reply:
That's a better phrasing!